security/backends/elasticsearch/connection.py

```diff
@@ -15,18 +15,23 @@ def connect(self, init_documents):
                 raise ImproperlyConfigured('You must set "SECURITY_ELASTICSEARCH_DATABASE" setting')
 
             connection.connection = connections.create_connection(
-                **settings.ELASTICSEARCH_DATABASE
+                **settings.ELASTICSEARCH_DATABASE,
+                timeout=settings.ELASTICSEARCH_TIMEOUT,
             )
             if init_documents:
                 self.init_documents()
 
     def init_documents(self):
         from security.backends.elasticsearch.models import (
-            InputRequestLog, OutputRequestLog, CommandLog, CeleryTaskRunLog, CeleryTaskInvocationLog
+            InputRequestLog, OutputRequestLog, CommandLog, CeleryTaskRunLog, CeleryTaskInvocationLog, PartitionedLog
         )
 
         for document in InputRequestLog, OutputRequestLog, CommandLog, CeleryTaskRunLog, CeleryTaskInvocationLog:
-            document.init()
+            if issubclass(document, PartitionedLog):
```

Comment:
viz níže, btw možná by bylo lepší přepsat ini na dokumentu, ať to není na dvou místech

Reply:
to issubclass je dost ošlivé. Možná by bylo lepší aby si o tomto rozhodoval Document (Elastic object) sám podle nastavení indexu. Tohle je takové ošklivé zalepování logiky.